BackEnd/blueprints/configurator/configurator.py

```diff
@@ -0,0 +1,52 @@
+from flask import Blueprint, request, jsonify
+from BackEnd.Functions.ImageModification.ColorChanging import ImageMod
+
+conf_bp = Blueprint('conf', __name__)
+
+
+@conf_bp.route('/test')
+def index():
+    return 'Test'
+
+
+@conf_bp.route('/process', methods=['POST'])
+def process():
+    try:
+        data = request.get_json()
+        if not data or 'image_base64' not in data or 'color_conf' not in data:
+            return jsonify({'error': 'Invalid data format'}), 400
+
+        processed_data = data_processing(data)
+        return jsonify({'processed_image': processed_data}), 200
+
+    except Exception as e:
+        return jsonify({'error': str(e)}), 500
+
+
+def data_processing(data):
```

Comment:
What data?

Change data to image_data or image_body
Change data_processing to image_data_processing